pystac_client/client.py

```diff
@@ -142,7 +143,9 @@ def get_all_items(self) -> Iterable["Item_Type"]:
         else:
             yield from super().get_items()
 
-    def search(self, **kwargs: Any) -> ItemSearch:
+    def search(self,
+               media_type: Optional[Union[str, pystac.MediaType]] = None,
+               **kwargs: Any) -> ItemSearch:
```

Comment:
```suggestion
    def search(self, **kwargs: Any) -> ItemSearch:
```
Same as for `Client.open`... I think it would be more appropriate to hard-code the `media_type` and remove this parameter.